python/scannerpy/table.py

```diff
@@ -37,13 +37,14 @@ def columns(self, index=None):
         return columns[index] if index is not None else columns
 
     def num_rows(self):
-        return self._descriptor.num_rows
+        return self._descriptor.end_rows[-1]
 
     def rows(self):
         if self._job is None:
             return list(range(self.num_rows()))
         else:
             if len(self._task.samples) == 1:
+                assert(False)
```

Comment:
I wasn't sure what you were trying to support here. You wanted to get the rows from the parent table?